services/core-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -111,18 +138,43 @@ export const renderDocumentLinkColumn = (
 export const renderTaggedColumn = (
   dataIndex: string,
   title: string,
-  tag: string,
   sortable = false,
   placeHolder = ""
 ) => {
   return {
     title,
     dataIndex,
     key: dataIndex,
-    render: (text: any) => {
-      const content = <div title={title}>{text ?? placeHolder}</div>;
-      return withTag(tag, content);
+    render: (text: any, record: any) => {
+      const content = (
+        <div
+          className={record.number_of_versions !== undefined ? "file-name-text" : ""}
+          style={record?.number_of_versions === 0 ? { marginLeft: "38px" } : {}}
+        >
+          {text ?? placeHolder}
+        </div>
+      );
+      return withTag(record, content, title);
     },
     ...(sortable ? { sorter: nullableStringSorter(dataIndex) } : null),
   };
 };
+
+export const actionOperationsColumn = (title = "", dataIndex = "operations") => {
```

Comment:
This one is also just for document actions, not reusable for actions in general, needs rename. I would like to see a function where we can pass in an array like [{actionTitle, actionFunction, actionIcon}], but that may be out of scope for this ticket.

Reply:
This has been renamed now as well